### DISCUSSION TITLE
[Bug Report] LastModifiedTimestamp on Listing objects is not updated if you simply delete an image from a listing and save

### DISCUSSION THREAD

**Discussion by amphelps535** [etsy/open-api#1294] [Q&A]
Query the API to get LastModifiedTimestamp long, note it, then go to etsy and simply delete 1 listing image from the given listing. You will find that LastModifiedTimestamp has not changed, even though the listing has, in fact, been modified by removing a listing image.

**Comment by showtroylove** [comment]
Check both last_modified_timestamp and updated_timestamp. Also, some image operations are asynchronous and queued for processing later. What you see on the web UI may not reflect the current state of the dataset. You must wait and then check the listing update timestamps. It's unclear why there are two seemingly similar timestamps, but I typically check both.
